terraform/modules/tf_stream_alert_app/README.md

```diff
@@ -0,0 +1,162 @@
+# StreamAlert App Integration Terraform Module
```

Comment:
Don't we want this to be a `.rst` file to be consistent with the rest of the files? GitHub renders `.rst` just fine

Reply:
`.rst` would also make the table a lot easier to understand in the raw code

Reply:
@austinbyers all of the TF readmes are md, so I followed that trend. Not changing it now :)

Reply:
Ok! Didn't know the TF stuff were READMEs